projects/ion/src/lib/smart-table/smart-table.component.spec.ts

```diff
@@ -120,6 +119,10 @@ describe('TableComponent', () => {
     expect(screen.getByText(label)).toBeInTheDocument();
   });
 
+  it('should not render a no data message', async () => {
+    expect(screen.queryByTestId('withoutData')).toBeNull();
```

Comment:
withoutData was removed, check by text with queryallbytext and tohavelenght 0 in expect